tensorflow/contrib/seq2seq/python/ops/helper.py

```diff
@@ -443,7 +443,7 @@ def sample(self, time, outputs, state, name=None):
     # Outputs are logits, use argmax to get the most probable id
     if not isinstance(outputs, ops.Tensor):
       raise TypeError("Expected outputs to be a single Tensor, got: %s" %
```

Comment:
Sure, I was actually borrowing the pattern from [right above it](https://github.com/critiqjo/tensorflow/blob/84f0cce/tensorflow/contrib/seq2seq/python/ops/helper.py#L337-L338). Should I change that too?